types/papaparse/index.d.ts

```diff
@@ -18,6 +19,8 @@ export function parse(file: File, config?: ParseConfig): ParseResult;
 
 export function parse(stream: ReadableStream, config?: ParseConfig): ParseResult;
 
+export function parse(stream: 1, config?: ParseConfig): ReadableStream;
```

Comment:
can you use `typeof NODE_STREAM_INPUT` here instead? I think it is easier to read.

Reply:
Makes sense